paper-drawer-panel.html

```diff
@@ -393,6 +396,22 @@
         this.transition = true;
       },
 
+      attached: function() {
+        this.listen(this.$.main, 'transitionend', '_transitionEndCallback');
+        this.listen(this.$.main, 'webkitTransitionEnd', '_transitionEndCallback');
+      },
+
+      detached: function() {
+        this.unlisten(this.$.main, 'transitionend', '_transitionEndCallback');
+        this.unlisten(this.$.main, 'webkitTransitionEnd', '_transitionEndCallback');
+      },
```

Comment:
I think you can remove the above `listen` and `unlisten` and just do `on-<event>` annotation syntax.  e.g.
```
<div id="main" on-transitionend="_transitionEndCallback">
```